android/MobiledgeXSDKDemo/matchingenginehelper/src/main/java/com/mobiledgex/matchingenginehelper/MatchingEngineHelper.java

```diff
@@ -1144,6 +1150,14 @@ public void postLocationToEdgeEvents(Location location) {
         }).start();
     }
 
+    public String getClosestCloudletHostname() {
+        if (mClosestCloudlet == null) {
+            return null;
+        } else {
+            return mClosestCloudlet.getFqdn();
+        }
+    }

```

Comment:
I think MatchingEngine has a  getHost(fcReply, port) method, but  getFqdn() output should be the same plus the domain name.